lib/galaxy/model/migrations/scripts.py

```diff
@@ -319,6 +212,19 @@ def run_upgrade(self, gxy_db_url=None, tsi_db_url=None):
         self._upgrade(gxy_db_url, GXY)
         self._upgrade(tsi_db_url, TSI)
 
+    def rename_config_argument(self, argv: List[str]) -> None:
+        """
+        Rename the optional config argument: we can't use '-c' because that option is used by Alembic.
```

Comment:
Is that still needed given that you're using the API now ?

Reply:
It could be, I'll check. But this might be used by manage_db.py, which is invoked used by the ansible role.

Reply:
Yep, it's used by the ansible role.